src/app/clusters/descriptor/descriptor.h

```diff
@@ -14,6 +14,8 @@
  *    limitations under the License.
  */
 
+#include <cstdint>
```

Comment:
Should this include go in the .cpp file instead?

Reply:
The header uses `uint16_t` so it needs this